_includes/cwl/metadata_example.cwl

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env cwl-runner
+
+class: CommandLineTool
+id: Example tool
+label: Example tool
+cwlVersion: v1.0
+doc: |
+    An example tool demonstrating metadata.
+
+requirements:
+  - class: ShellCommandRequirement
+
+inputs:
+  bam_input:
+    type: File
+    doc: The BAM file used as input
+    format: http://edamontology.org/format_2572
+    inputBinding:
+      position: 1
+
+stdout: output.txt
+
+outputs:
+  report:
+    type: File
+    format: http://edamontology.org/format_1964
```

Comment:
I guess that you prefer this format for namespaces in all the `.cwl` files? And what about the `sample.yml` input file, which also includes a `format: http://edamontology.org/...` line?